### DISCUSSION TITLE
Request app-ads.txt static file inside public folder always return 404?

### DISCUSSION THREAD

**Discussion by RafilAlArsApps** [vercel/vercel#5680] [Help]
Created an app-ads.txt file that includes the authorized sellers for my google ads (it's required by google AdMob), then publish it at mydomain.com/app-ads.txt always return 404, when I changed the file name to something else working but with this name never work I do not know why that happening anybody could help please.

I put the file inside public folder.

**Comment by paulogdm** [comment]
Closing in favor of #5679
